staging/cse/windows/provisioningscripts/kubeproxystart.ps1

```diff
@@ -50,13 +50,6 @@ if ($Global:ClusterConfiguration.Kubernetes.Kubeproxy.ConfigArgs) {
     $global:KubeproxyArgList += $Global:ClusterConfiguration.Kubernetes.Kubeproxy.ConfigArgs
 }
 
-$hnsNetwork = Get-HnsNetwork | ? Name -EQ $KubeNetwork
```

Comment:
No concern to remove wait. But for cleanup hnspolicylist, I think that we need to double confirm it with upstream team and windows team.

Reply:
Move this change to https://github.com/Azure/AgentBaker/pull/4131